<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInput</class>
 <widget class="QWidget" name="FormInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OutDataBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OutReportBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="EChartsBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
